<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>1281</width>
      <height>850</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="aiTitle">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>341</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 154, 6);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;HOLAdwa&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#cc0000;&quot;&gt;TEst!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionHelpWithMove"/>
    <addaction name="actionRestart"/>
    <addaction name="actionDevMode"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionShowMask"/>
    <addaction name="separator"/>
    <addaction name="actionShowFreeTow"/>
    <addaction name="actionShowFreeThree"/>
    <addaction name="actionShowFreeFour"/>
    <addaction name="actionShowWin"/>
    <addaction name="actionShowCapture"/>
    <addaction name="actionShowUnderCapture"/>
    <addaction name="actionShowTowFreeThree"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDevMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dev Mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionShowMask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Mask</string>
   </property>
  </action>
  <action name="actionPvPMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PvP Mode</string>
   </property>
  </action>
  <action name="actionShowFreeTow">
   <property name="text">
    <string>Show Free Tow</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionShowFreeThree">
   <property name="text">
    <string>Show Free Three</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionShowFreeFour">
   <property name="text">
    <string>Show Free Four</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionShowWin">
   <property name="text">
    <string>Show Win</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionShowCapture">
   <property name="text">
    <string>Show Capture</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionShowUnderCapture">
   <property name="text">
    <string>Show Under Capture</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionShowTowFreeThree">
   <property name="text">
    <string>Show Tow Free Three</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionHelpWithMove">
   <property name="text">
    <string>Help With Move</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
